### PR TITLE
Provide System.Composition.AttributedModel package readme

### PR DESCRIPTION
**Pull request by eNeRGy164**
Fill out details for the `System.Composition.AttributedModel` NuGet package readme.

See https://github.com/dotnet/runtime/issues/99358 and https://github.com/dotnet/runtime/pull/106443

### CODE REVIEW

**Review by ericstj — APPROVED** [review]
LGTM, @steveharter / @buyaa-n - can one of you have a look?

File: src/libraries/System.Composition/src/System.Composition.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <NoWarn>$(NoWarn);NU5128</NoWarn>
     <!-- Reference dependencies to persiste them in the project's project.assets.json file which is used to calculate package dependencies. -->
     <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
-    <PackageDescription>Provides a version of the Managed Extensibility Framework (MEF) that is lightweight and specifically optimized for high throughput scenarios, such as the web.</PackageDescription>
+    <PackageDescription>Provides the foundational attributes that allow you to declare parts for composition, such as imports, exports, and metadata with the Managed Extensibility Framework (MEF).</PackageDescription>
```
Comment (buyaa-n) [inline src/libraries/System.Composition/src/System.Composition.csproj:11]: Isn't it supposed to be added/updated for System.Composition.AttributedModel.csproj?

https://github.com/dotnet/runtime/blob/f2d5480e5f1e0ade3e73cdd176151f625d6db278/src/libraries/System.Composition.AttributedModel/src/System.Composition.AttributedModel.csproj#L8-L18

Comment (eNeRGy164) [reply src/libraries/System.Composition/src/System.Composition.csproj:11]: You're right, something must have gone wrong. Will update this.

**Review by buyaa-n — APPROVED** [review]
Thank you!

**Comment by buyaa-n** [comment]
/ba-g remaining test failure unrelated and has no definitive log to file an issue

**Pull request merged**